kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -272,6 +276,16 @@ export default function useQuizCreation(DEBUG = false) {
       setActiveSection(newSection.section_id);
     }
     _fetchChannels();
+
+    // Set watcher once we have a section in place
+    watch(activeResourcePool, (resourcePool, old) => {
```

Comment:
Presumably we initiate this whenever we do, so we might be able to decide more directly when to do this rather than a potentially expensive or unreliable isEqual call?

Not clear that this is actually a problem, so definitely not a blocker.